src/tools/rustfmt/ci/integration.sh

```diff
@@ -53,67 +54,72 @@ function check_fmt_base {
         return 1
     fi
     cat rustfmt_output
-    ! cat rustfmt_output | grep -q "internal error"
-    if [[ $? != 0 ]]; then
+    grep -q "internal error" < rustfmt_output
+    cmd_ret=$?
```

Comment:
Lint was complaining that we were using `$?` and that it could be problematic since the last run command might not be the one expected. So instead, we store the value in a variable and then use this variable.